nixos/doc/manual/man-pages.xml

```diff
@@ -6,7 +6,7 @@
   <author><personname><firstname>Eelco</firstname><surname>Dolstra</surname></personname>
    <contrib>Author</contrib>
   </author>
-  <copyright><year>2007-2020</year><holder>Eelco Dolstra</holder>
+  <copyright><year>2007-2022</year><holder>Eelco Dolstra</holder>
```

Comment:
```suggestion
  <copyright><year>2007-2022</year><holder>NixOS contributors</holder>
```

This is a community effort where many have contributed. To illustrate:

```
[nixpkgs/nixos/doc/manual]$ for f in man-*; do git blame $f; done | grep Eelco | wc -l
593

[nixpkgs/nixos/doc/manual]$ for f in man-*; do git blame $f; done | wc -l
1875

```

While these stats may be biased, no copyright was reassigned, to my knowledge, and we can't trivialize all contributions by others.

Reply:
I think "the NixOS Contributors" (with a lowercase "the") would make more sense in the context, or am I wrong here? Iirc this is only used in the last line of each man page

Reply:
I don't think the group of people is definitive enough to use the definitive article. If the phrasing evokes the question "which NixOS contributors?", that's probably a good thing. I'm neither a native speaker or lawyer though.

Reply:
cc @edolstra

Reply:
`COPYING` says "Eelco Dolstra and the Nixpkgs/NixOS contributors". Legally it's a bit tautological since it basically just says that it's copyrighted by the people who hold the copyright...